lib/events.py

```diff
@@ -43,7 +43,7 @@
 APPEAL_RECEIVE = "appeal_receive"
 APPEAL_DISPOSITION = "appeal_disposition"
 
-UOF_INCIDENT = "uof_incident"
+UOF_INCIDENT = "uof_occur"
```

Comment:
We might need to communicate this change with Huy

Reply:
Got it. On it after I review the changes.